packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -107,44 +106,11 @@ export class ComponentPageBase extends React.PureComponent<IComponentPageProps>
     );
   }
 
-  private _getNativePropsInfo(): JSX.Element | undefined {
```

Comment:
Wondering if we should still show this if the page uses the old style of property table?

Reply:
If the extends isn't available in certain cases then I think that makes sense as it provides valuable information.